app/graphql/schema/queries/customer-query-by-ticket-id.js

```diff
@@ -0,0 +1,67 @@
+const cosmos = require('../../../cosmos')
+const { cosmosConfig } = require('../../../config')
+const { convertCosmosTimestamp } = require('../../../utils')
+
+const customerQueryByTicketId = async (_root, args, context) => {
+  try {
+    const { queriesDatabase } = await cosmos()
+
+    const responseQuerySpec = {
+      query: 'SELECT * FROM customerQueryResponse cq WHERE cq.ticketId = @ticketId ORDER BY cq._ts DESC',
+      parameters: [{ name: '@ticketId', value: `${args.ticketId}` }]
+    }
+
+    const response = await queriesDatabase
+      .container(cosmosConfig.queriesContainer)
+      .items.query(responseQuerySpec)
+      .fetchAll()
+
+    if (!args.ticketId) {
+      throw new Error('ticketId is must be provided')
+    }
+
+    const originalQuerySpec = {
+      query: 'SELECT * FROM customerQueryResponse cq WHERE cq.ticketId = @ticketId',
+      parameters: [{ name: '@ticketId', value: `${args.ticketId}` }]
+    }
+
+    const originalQueryResponse = await queriesDatabase
+      .container(cosmosConfig.queriesContainer)
+      .items.query(originalQuerySpec)
+      .fetchAll()
+
+    if (!originalQueryResponse.resources.length) {
+      throw new Error(`No customer query data found for ticketId ${args.ticketId}`)
+    }
+
+    const originalQuery = originalQueryResponse.resources[0]
+
+    return {
+      code: 200,
```

Comment:
We return the code/sucess/message here twice

Reply:
Think the issue I'm having in the [comment below](https://github.com/DEFRA/ffc-sfd-data/pull/15#discussion_r1653180539) also applies here?